x/incentive/keeper/keeper_fees.go

```diff
@@ -0,0 +1,31 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+)
+
+// Move gas fees collected to incentive module
+func (k Keeper) CollectGasFeesToIncentiveModule(ctx sdk.Context) sdk.Coins {
+	// fetch and clear the collected fees for distribution, since this is
+	// called in BeginBlock, collected fees will be from the previous block
+	// (and distributed to the previous proposer)
+	feeCollector := k.authKeeper.GetModuleAccount(ctx, k.feeCollectorName)
+	feesCollectedInt := k.bankKeeper.GetAllBalances(ctx, feeCollector.GetAddress())
+	// feesCollected := sdk.NewDecCoinsFromCoins(feesCollectedInt...)
```

Comment:
Can you remove this commented code?

Reply:
Okay got it. Will do this in the next PR.